### ISSUE TITLE
crashes constraints when body is removed

### ISSUE THREAD

**Issue by ken-sands** [schteppe/p2.js#230]
when a body that makes up a constraint is removed the constraint should remove or at least check for non null bodies and just return. currently removing a body that is part of a constraint causes a crash

**Comment by schteppe** [comment]
Hi.
Yeah, that's a good idea. I'm thinking of just removing the constraints from the world automatically when one of the bodies are removed, like in box2d.

**Comment by ken-sands** [comment]
makes sense. I see no reason for the constraint to exist once either of the connected bodies are removed.
